config/environment_config.yml

```diff
@@ -14,6 +14,7 @@ local:
   admin_url: http://admin.et.127.0.0.1.nip.io:3100/admin
   sidekiq_url: http://et1.et.127.0.0.1.nip.io:3100/apply/sidekiq
   mailhog_url: http://mail.et.127.0.0.1.nip.io:3100/api/v2/messages
+  mailhog_search_url: http://mail.et.127.0.0.1.nip.io:3100/api/v2/messages
```

Comment:
This is the same as mailhog_url ?

Reply:
no, it should be http://mail.et.127.0.0.1.nip.io:3100/api/v2/search